apis/xnetworks/definition.yaml

```diff
@@ -27,13 +28,6 @@ spec:
                     region:
                       type: string
                       description: Region is the region you'd like your resource to be created in.
-                    deletionPolicy:
```

Comment:
we removed deletionPolicy - but we can get the same with `managementPolicies`, so i wonder if we want to add this instead

Orphan: managementPolicies: ['Observe', 'Create', 'Update', 'LateInitialize']
Delete: managementPolicies: ['Observe', 'Create', 'Update', 'LateInitialize', 'Delete']